hypertrace-core-graphql-attribute-store/src/main/java/org/hypertrace/core/graphql/attributes/AttributeModelTranslator.java

```diff
@@ -57,7 +80,33 @@ public Optional<AttributeModel> translate(AttributeMetadata attributeMetadata) {
     }
   }
 
-  @SuppressWarnings("unused")
+  public Optional<AttributeMetadata> translate(final AttributeModel attributeMetadata) {
```

Comment:
All the various attribute containers confuse me, but this is going rom the gql model to the attribute service proto on the create path right? So we shouldn't have an ID, be setting custom etc. Did we let the client accept a fully identified proto in the create API?

Reply:
Sorry, I didn't get what you mean by 
> Did we let the client accept a fully identified proto in the create API?

Reply:
Typically on a create API, you accept some subset of fields, and then the service will generate the rest (like an ID) and return them in the response. Here we're asking the client to provide the whole object - including the ID - which is a problem

Reply:
Got it! Updated.